compose/project.py

```diff
@@ -776,13 +776,13 @@ def get_secrets(service, service_secrets, secret_defs):
                 .format(service=service, secret=secret.source))
 
         if secret_def.get('external'):
-            log.warn("Service \"{service}\" uses secret \"{secret}\" which is external. "
+            log.warning("Service \"{service}\" uses secret \"{secret}\" which is external. "
                      "External secrets are not available to containers created by "
```

Comment:
Here a new indentation is necessary. 
What `flake8` is telling you is that the string parts that follow the line 779  have to start at the same column as the first one.

Reply:
I see, I this is the only one which isn't properly aligned.

Reply:
Is there any way to know flake8's linting? 
Moreover, this line seems properly aligned on my PC. :/